iot_config/src/org.rs

```diff
@@ -142,6 +142,26 @@ pub async fn get_with_constraints(
     })
 }
 
+pub async fn toggle_status(
```

Comment:
Nit: Maybe call this set status? Toggle implies turning on or off based on the previous state